api.planx.uk/.husky/pre-commit

```diff
@@ -2,3 +2,5 @@
 . "$(dirname -- "$0")/_/husky.sh"
 cd api.planx.uk
 pnpm dlx lint-staged
+cd ../e2e
+pnpm dlx lint-staged
```

Comment:
I've found that I've needed to run `pnpm run lint:fix` on every commit from the `e2e` directory so hopefully this fixes that

Reply:
I'm not sure this is the right solution - this seems to be running e2e linting each time a commit is made in the api directory. There should be a pre-commit file in the e2e directory which does this (clearly it's not working though!)

I think we need to actually install lint-staged in the root ([docs](https://github.com/lint-staged/lint-staged?tab=readme-ov-file#how-to-use-lint-staged-in-a-multi-package-monorepo)) and then we can have a per-project husky working as expected?